hamlet/theses/forms.py

```diff
@@ -85,16 +69,15 @@ class Meta:
 
 
 class UploadFileForm(forms.Form):
-    allowed_extensions = ['txt']
-    allowed_mimetypes = ['text/plain']
-    allowed_filetypes = ['text/plain']
+    allowed_extensions = ['txt', 'docx']
+    allowed_mimetypes = ['text/plain',
+        'application/vnd.openxmlformats-officedocument.wordprocessingml.document']
```

Comment:
Do you need a file generated from an actual Windows product to test this on? I can easily supply you with an OS X Office file, and I can get a Windows one also.

Reply:
Probably wouldn't hurt, though I'd guess with what we're doing, any differences in implementation of the standard won't have much of an impact. It's no problem to throw a few more files at it during testing, so why not?

Reply:
I've sent you a Mac-generated one, and Matt will be sending you a Windows-generated one shortly.